src/Grpc.AspNetCore.Server/Internal/CallHandlers/ClientStreamingServerCallHandler.cs

```diff
@@ -31,14 +33,25 @@ internal class ClientStreamingServerCallHandler<TService, TRequest, TResponse> :
         where TService : class
     {
         private readonly ClientStreamingServerMethod<TService, TRequest, TResponse> _invoker;
+        private readonly IEnumerable<Type> _interceptors;
```

Comment:
I'd prefer to store a list or array over an enumerable.

Reply:
I'm thinking we'll go with a custom collection.